docs/proposals/multi-platform.md

```diff
@@ -0,0 +1,451 @@
+# Kubernetes for multiple platforms
+
+**Author**: Lucas Käldström ([@luxas](https://github.com/luxas))
+
+**Status** (05/06/2016): Most parts are already implemented; still there's room for improvement
```

Comment:
minor nit: I can't tell if this date is US or rest-of-world format. Maybe use YYYY-MM-DD to be clear?